src/instructlab/eval/mmlu.py

```diff
@@ -1,24 +1,30 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Local
-from .evaluator import Evaluator
+from instructlab.eval.evaluator import Evaluator
 
+# Third Party 
+from lm_eval.evaluator import simple_evaluate
+import os
 
 class MMLU_Evaluator(Evaluator):
     """
     Child class of an Evaluator for Massive Multitask Language Understanding (MMLU)
 
     Attributes:
+        model_path   absolute path to or name of a huggingface model
```

Comment:
I changed model_path to model_name in #9.  If you are going to need model_path, we'll likely need to remove it from the Evaluator parent class.  I guess the question is who will be doing the serving.

Reply:
@alinaryan unless there's a reason for divergence I think we should stay consistent if possible and use `model_name`

Reply:
`model_path` here is used differently than in MT-Bench. `model_path` is can be the path on the filesystem to the locally downloaded model. This model can be downloaded via `ilab download --repository=instructlab/granite-7b-lab`.

The `model_path` can also be the name of the repo and the model on huggingface. For example just passing ing `instructlab/granite-7b-lab` would download the model from https://huggingface.co/instructlab/granite-7b-lab.

If we want as a follow up maybe we ensure that the `model_path` is only for a model downloaded locally, but I don't think that should hold up this PR.

Hope this clarity helps on why the name is as is.

Reply:
My point was that if we do continue to need a full model path here, we should just remove it from Evaluator since it's not common between mtbench and mmlu.

Reply:
Yeah it looks like we'll need to remove it from the Evaluator base class. Should we do that in this PR?

Reply:
That works.  I'll make the corresponding adjustment in #9.

Reply:
I agree about staying consistent, but it does need to be model_path bc that's what the CLI expects.

Reply:
I didn't see the above discussion until after I posted my comment, but this sounds good! I'll remove it from the base class too

Reply:
Sounds good y'all!